src/app/components/glamourform/inputs/ChestInputs.tsx

```diff
@@ -0,0 +1,72 @@
+import { useAppContext } from "../../context/AppContext";
+
+export default function ChestInputs() {
+  const { chest } = useAppContext();
+
+  function displayError() {
+    if (
+      !chest.invalidPiece &&
+      !chest.invalidPieceDye &&
+      !chest.invalidPieceDyeGroup
+    )
+      return <></>;

```

Comment:
Do we need <></>? Can we just end the first check with return?

Reply:
Removed the React fragment and function is able to be ended with just the return.

![image](https://github.com/brandon-moy/glamourngpicker/assets/109567499/81304120-ace3-458a-b442-2f3422781376)

![image](https://github.com/brandon-moy/glamourngpicker/assets/109567499/3b2701dd-7f4a-4a8d-91a7-c8e43eaf2aba)